diffexpr/py_deseq.py

```diff
@@ -92,6 +92,10 @@ def run_deseq(self, **kwargs):
             BPPARAM = bpparam()
         )
         """
+        
+        for arg in kwargs.keys():
+            if arg == 'reduced':
+                kwargs[arg] = Formula(kwargs[arg])
```

Comment:
```suggestion
        for key, value in kwargs.items():
            if key == 'reduced':
                kwargs[key] = Formula(value)
```

just a minor suggestion, but I think using ```.item``` is cleaner when we modify a dictionary

Reply:
Alright :)